dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -74,6 +76,12 @@
 
   private final ImmutableList<Path> inputClassPath;
   private final Repository classRepository;
+
+  // Sometimes classes are not placed in the root of a JAR file. For example, Spring Boot Gradle
+  // Java plugin places class files under "BOOT-INF/classes". ClassDumper needs to remember the
```

Comment:
https://docs.spring.io/spring-boot/docs/current/reference/html/executable-jar.html

Reply:
Updated the URL